bin/run_clang_tidy.sh

```diff
@@ -2,7 +2,7 @@
 
 set -e
 
-BUILD_PATH=/build/faabric/
+BUILD_PATH=/build/faabric/static/
```

Comment:
Building staticallly is the default, so we assume this in development environments.